DOCS/man/input.rst

```diff
@@ -532,7 +532,7 @@ Remember to quote string arguments in input.conf (see `Flat command syntax`_).
         Whether to run the process in detached mode (optional, default: no). In
         this mode, the process is run in a new process session, and the command
         does not wait for the process to terminate. If neither
-        ``capture_stdout`` nor ``capture_stderr`` have been set to ``yes``,
```

Comment:
do we really want to remove the formatting of the former yes here?

Reply:
I'm not quite sure I understand, do you want me to use \`true\`? I didn't quote it because it wasn't quoted in all the existing mentions. If we quote it here we should quote it everywhere.

Reply:
it was merely a question if it would make sense to quote them too? though i guess no from the explanation in your other comment.

Reply:
I think it would be clearer to highlight true and false. That's not necessarily related to this PR in my opinion since we should quote every existing mention across the manpage, but I'm willing to do it if the other maintaners agree.